packages/extensionmanager/src/model.ts

```diff
@@ -176,6 +186,10 @@ export class ListModel extends VDomModel {
       entries[pkg.name] = {
         name: pkg.name,
         description: pkg.description,
+        link:
+          'homepage' in pkg.links
+            ? pkg.links.homepage
+            : 'repository' in pkg.links ? pkg.links.repository : pkg.links.npm,
```

Comment:
Similarly, I do not know if `homepage` is mandatory so the code tries `homepage`, `repository`, then `npm`.